docs/models.rst

```diff
@@ -69,7 +69,9 @@ Further, let's take a look at created fields for models
 
     id = fields.IntField(pk=True)
 
-This code defines integer primary key for table. Sadly, currently only simple integer pk is supported.
+This code defines integer primary key for table.
+If you don't define a primary key, we will create a Integer primary key with name of ``id`` for you.
+Sadly, currently only simple integer primary key is supported.
```

Comment:
May be should force using `'id` field at the moment? Current inner implementations using `id` all over the place and setting another name for pk will fall everything apart.
It made me realise why django needed separate `pk` field that stores reference to pk field

Reply:
I was having similar thoughts when I did this.
The reason I forced a default `id` fields was because of all the internal code using `.id` everywhere.

Reply:
Or should we copy Django, and have a `pk` field, and whatever has `pk=True` is just an alias to `pk`?
Or should we just stick to `id`?

Reply:
Then we should fix this description for now, because it's confusing and gives hope that you can use different name for pk. I created issue regarding this https://github.com/Zeliboba5/tortoise-orm/issues/36 and hope we will fix it before 1.0 because it's quite important in my opinion.
Could you also add some kind of validation that none other fields were defined with `pk=True` while we are forcing it ourselves